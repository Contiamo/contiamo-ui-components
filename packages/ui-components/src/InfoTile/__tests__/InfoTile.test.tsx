import * as React from "react"
import { render } from "enzyme"

import ThemelessInfoTile from "../InfoTile"
import wrapDefaultTheme from "../../../utils/wrap-default-theme"

const InfoTile = wrapDefaultTheme(ThemelessInfoTile)

describe("InfoTile", () => {
  it("Should render", () => {
    const renderedComponent = render(<InfoTile label="Country">DE</InfoTile>)
    expect(renderedComponent).toMatchSnapshot()
  })
})
