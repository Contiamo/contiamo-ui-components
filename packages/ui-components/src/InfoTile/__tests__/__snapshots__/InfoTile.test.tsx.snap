// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoTile Should render 1`] = `
<div
  class="css-farlai"
>
  <small
    class="css-11y8e93"
  >
    Country
  </small>
  <span>
    DE
  </span>
</div>
`;
